v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -397,26 +397,18 @@ export const GraphDataConfigurationModel = DataConfigurationModel
         }
       })
     },
-    binDetails(_binAlignment?: number, _binWidth?: number, reset = false) {
+    binDetails(_binAlignment?: number, _binWidth?: number) {
       const kDefaultNumberOfBins = 4
```

Comment:
I wonder if this method belongs better in `GraphContentModel` where the `binWidth` and `binAlignment` are defined?

Reply:
Yeah, I think that makes sense. I originally added it to the data configuration model because it uses a bunch of data configuration model properties, but I just realized we can easily get those in the graph model via its `dataConfiguration` property. So I've moved it there in the latest commit.